src/cpp/session/modules/rmarkdown/SessionRMarkdown.cpp

```diff
@@ -1372,6 +1384,11 @@ Error initialize()
    methodDef.numArgs = 1;
    r::routines::addCallMethod(methodDef);
 
+   r::routines::registerCallMethod(
```

Comment:
nit: use `RS_REGISTER_CALL_METHOD()` macro here (avoids need to specify number of arguments)